src/main/java/io/ballerina/plugins/idea/configuration/ui/BallerinaSdkSelection.java

```diff
@@ -0,0 +1,310 @@
+/*
+ * Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com).
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package io.ballerina.plugins.idea.configuration.ui;
+
+import com.intellij.openapi.fileChooser.FileChooser;
+import com.intellij.openapi.fileChooser.FileChooserDescriptor;
+import com.intellij.openapi.project.Project;
+import com.intellij.openapi.ui.ComboBox;
+import com.intellij.openapi.vfs.VirtualFile;
+import com.intellij.ui.JBColor;
+import com.intellij.util.ui.JBUI;
+import io.ballerina.plugins.idea.sdk.BallerinaSdkService;
+import io.ballerina.plugins.idea.sdk.BallerinaSdkSettings;
+import io.ballerina.plugins.idea.sdk.BallerinaSdkUtil;
+
+import java.awt.Color;
+import java.awt.Font;
+import java.awt.GridBagConstraints;
+import java.awt.GridBagLayout;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.Objects;
+
+import javax.swing.JCheckBox;
+import javax.swing.JComponent;
+import javax.swing.JLabel;
+import javax.swing.JPanel;
+
+/**
+ * Provides UI for selecting a global sdk version for Intellij IDEA.
+ * Can use system sdk and custom sdk and switch between them.
+ *
+ * @since 2.0.0
+ */
+public class BallerinaSdkSelection {
```

Comment:
```suggestion
public class BallerinaSettingsPanel {
```

Reply:
The `BallerinaLanguageSettingsConfigurable` class is designed to add custom UI components to the settings, such as the SDK selection component. Therefore, the `BallerinaSdkSelection` class is used specifically to add a UI component for SDK selection to the language settings page. Renaming it to `BallerinaSettingsPanel` would be misleading because this class does not represent the entire settings page.